src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -109,18 +124,23 @@ async function get_config_data_if_exists(config_root_backend, config_file_path,
 /**
  * get_bucket_owner_account will return the account of the bucket_owner
  * otherwise it would throw an error
- * @param {object} global_config
- * @param {string} bucket_owner
+ * @param {Object} global_config
+ * @param {string} dir_path directory with account file (either accounts, root_accounts or access_keys)
+ * @param {string} root_account_identifier root account file name, either name or id
+ * @param {boolean} is_symlink whether while is symlink or not. Name -> true, id -> false/undef
  */
-async function get_bucket_owner_account(global_config, bucket_owner) {
-    const account_config_path = get_config_file_path(global_config.accounts_dir_path, bucket_owner);
+async function get_bucket_owner_account(global_config, dir_path, root_account_identifier, is_symlink) {
+    //for root accounts, iam_account is optional and is equal to root_account_identifier
+    const account_config_path = is_symlink ?
+        get_symlink_config_file_path(dir_path, root_account_identifier, root_account_identifier) :
+        get_config_file_path(dir_path, root_account_identifier);
     try {
         const account = await get_config_data(global_config.config_root_backend, account_config_path);
         return account;
     } catch (err) {
         if (err.code === 'ENOENT') {
-            const detail_msg = `bucket owner ${bucket_owner} does not exists`;
-            throw_cli_error(ManageCLIError.BucketSetForbiddenBucketOwnerNotExists, detail_msg, {bucket_owner: bucket_owner});
+            const detail_msg = `bucket owner ${root_account_identifier} does not exists`;
```

Comment:
If we're trying to validate bucket, we are trying to get the account.
If we fail, it means the bucket owner json is missing (either because bucket json is wrong or account json file is not there).
Under such circumstance, this message is best we can do and is better than nothing.

Reply:
Thanks, could you please attach this explanation as a comment?